src/components/shared/LinksProfile.tsx

```diff
@@ -0,0 +1,13 @@
+import Link from 'next/link'
+
+const LinksProfile = ({text}) => {
+    return (
+
+        <Link href="#" className="block hover:bg-gray-100 px-4 py-2 text-gray-700 text-sm">
+                {text}
+        </Link>
+
+    )
+}
+
```

Comment:
```suggestion
const LinksProfile = ({text, url}) => {
    return (

        <Link href={url} className="block hover:bg-gray-100 px-4 py-2 text-gray-700 text-sm">
                {text}
        </Link>

    )
}

```

creo que te falto añadir esta prop para que sea completamente reutilizable